spec/unit/artifactserver_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'spec_helper'
 require 'cookbook-omnifetch/artifactserver'
+require 'zlib'
+require 'archive/tar/minitar'
```

Comment:
I think it'd be cool to have a standardized way to build tarballs for any tool that ships to supermarket. I would also use it in the `chef export -a` (export archive) feature. But for the purposes of this PR, no need to block.